src/ListTasks1/OneLinkedList.c

```diff
@@ -0,0 +1,117 @@
+#include "OneLinkedList.h"
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+typedef struct ListNode ListNode;
+
+struct ListNode {
+    int value;
+    struct ListNode* next;
+};
+
+struct List {
+    ListNode* head;
+};
+
+List* newList()
+{
+    List* list = calloc(1, sizeof(*list));
+    return list;
+}
+
+bool isEmpty(List* list) { return list->head == NULL; }
```

Comment:
Желательно чтобы все методы были в одном стиле
```suggestion
bool isEmpty(List* list) {
    return list->head == NULL;
}
```